docs/admin/how-to/install/register-a-vo.md

```diff
@@ -1,5 +1,19 @@
 # Register a VO
 
+Registering a VO means allowing users of a specific Virtual Organization to use DiracX services. In practice, this can be done once an external IdP is used for managing such VO.
```

Comment:
This doesn't belong here, all VOs should be added as part of the v9 upgrade instructions.

Adding an entry to DisabledVOs will completely break said VO after completing the upgrade.

Reply:
Then this should be explained in DIRAC, it has no effect on DiracX.